src/components/o-pagination/index.scss

```diff
@@ -0,0 +1,93 @@
+@import "../../styles/settings/colors";
+@import "../../styles/mixins/respond";
+@import "../../styles/typo/typo-mixins";
+
+$pagination__link-color: $color-prim-gray-dusty !default;
+$pagination__link-color--hover: $color-prim-red-burnt-sienna !default;
+$pagination__link-width: 30px !default;
+$pagination__link-width-sm: 22px !default;
+$pagination__link-height: 20px !default;
+$pagination__link-height-sm: 18px !default;
+
+$pagination__link-border-width: 14px !default;
+$pagination__link-border-width-sm: 10px !default;
+$pagination__link-border-height: 3px !default;
+$pagination__link-border-margin: 6px !default;
+
+.o-pagination__root {
+  display: block;
+
+  // IMPORTANT: make sure to deal with inherited CSS properties here, like text-align!
+}
+
+.o-pagination {
+  text-align: center;
+}
+
+.o-pagination__first {
+
+}
+
+.o-pagination__last {
+
+}
+
+.o-pagination__first,
+.o-pagination__last,
+.o-pagination__link {
```

Comment:
@LucaMele 
Basically we could pick from 3 approaches to center and horizontally align this component:
- display: inline and text-center
- Floats and margin
- Flex

I think as of now the first the simplest and sufficient 🤔